src/index.html

```diff
@@ -129,30 +129,32 @@ <h1 class="ui header">Put your COVID certificate in your iPhone’s wallet.</h1>
                   </div>
                   <div class="extra content" id="cardExtraContent">
                   </div>
-                  <button id="saveInWallet" class="ui bottom attached primary big button"><i class="download icon"></i>Save in your wallet</button>
+                  <button type="button" id="saveInWallet" class="ui bottom attached primary big button"><i class="download icon" role="presentation"></i>Save in your wallet</button>
               </div>
-              <button class="ui fluid icon button" name="startScanning">
-                  <i class="qrcode icon"></i>
+              <button type="button" class="ui fluid icon button big" name="startScanning">
+                  <i class="qrcode icon" role="presentation"></i>
                   Start scanning
               </button>
-              <button class="ui fluid basic icon button" name="scanImage">
-                  <i class="file image icon"></i>
+              <button type="button" class="ui fluid basic icon button big" name="scanImage">
+                  <i class="file image icon" role="presentation"></i>
                   Import picture
               </button>
           </section>
 
-          <section id="faq" class="ten wide column">
-              <h3 class="ui center aligned header">
-                  FAQ
+          <div class="one wide column"></div>
```

Comment:
Should it be displayed only on a computer to prevent stacking on mobile? Using [visibility classes](https://semantic-ui.com/collections/grid.html#device-visibility) like `computer only` ?